src/operations/extrusions/extrudeRectangular.js

```diff
@@ -0,0 +1,44 @@
+const flatten = require('../../utils/flatten')
+
+const { geom2, path2 } = require('../../geometry')
+
+const extrudeRectangularPath2 = require('./extrudeRectangularPath2')
+const extrudeRectangularGeom2 = require('./extrudeRectangularGeom2')
+
+/**
+ * Extrude the given object(s) by following the outline(s) with a rectangle.
+ * @param {Object} options - options for extrusion, if any
+ * @param {Number} [options.radius=1] - radius of the rectangle
+ * @param {Integer} [options.segments=0] - number of segments for rounded ends, or zero for chamfer
```

Comment:
ha, we are meeting the chamfer vs rounded vs etc issue again ?

Reply:
Kind of, as expand is being used by this function. So, the options for expand also apply here. I adjusted the documentation.

By the way, I think 'radius' should become 'size' now. Rectangles are defined by 'size' now.